test_example.py

```diff
@@ -0,0 +1,2 @@
+def test_placeholder():
```

Comment:
this file shouldn't be toplevel

Reply:
I will create `API/pytest` folder and move the file then. I'm not sure what is the correct place for it.

Reply:
The tests I have put in this PR are **django** tests which are not detected by `pytest`, unfortunately. I don't really know why do we have to use `pytest` in the first place, since django has built-in tests. I'll do the `API/test` thing

Reply:
We use pytest as it's the go-to testing module for python in general. We have 2 options here. We can either run `python3 manage.py tests` in our `tests.yml` or install pytest plugin for django. We can have this conversation in our next meeting

Reply:
Sure thing @kapiw04